app/models/container_image.rb

```diff
@@ -82,5 +82,14 @@ def has_compliance_policies?
     !plist.blank?
   end
 
+  def annotate_deny_execution(causing_policy)
```

Comment:
> But, since ContainerImage does not yet support STI, add an error handler here in case someone calls this method directly. Stopping calls via MiqAction seems insufficient to protect this method.

@moolitayer can you take care of that? Thanks.